src/errors/test-run/templates.js

```diff
@@ -187,9 +187,15 @@ export default {
     `),
 
     [TYPE.actionIncorrectKeysError]: err => markup(err, `
-         The <code>${err.argumentName}</code> argument contains an incorrect key or key combination.
+        The <code>${err.argumentName}</code> argument contains an incorrect key or key combination.
 
-         ${err.getCallsiteMarkup()}
+        ${err.getCallsiteMarkup()}
+    `),
+
+    [TYPE.hybridFnExecutingInterruptedError]: err => markup(err, `
+        Client code execution was interrupted by page unload.
```

Comment:
> maybe we should figure out scenarios when it could happen

Agree! If would be supercool if we could!

> I'd rather say by page load

We just thought that it's technically `unload` that interrupts the client code. In any case, I wouldn't say it's `load` 'cause... It literally implies that the code was being executed while the page was **not loaded**, then at some moment the page **began loading**, which interrupted the execution. Which is nonsense.

Anyway, page load is not the prime cause. It was either triggered by reload of the same page or by navigating away. 

The question is - can we restrict *navigating away* to *redirection*? If I remember the discussion, all examples were about redirects.

If so, we can say, `by page reload or redirect`.